stories/topics/proc-aws-upgrade-pull-container-image.adoc

```diff
@@ -1,13 +1,13 @@
 [id="proc-aws-upgrade-pull-container-image"]
 
-= Pulling the ansible-on-clouds-ops 2.3 container image
+= Pulling the ansible-on-clouds-ops 2.4 container image
 
 .Procedure
 * Pull the Docker image for Ansible on Clouds operational container with the same tag as the version you want to upgrade to.
 +
 [NOTE]
 ====
-Before pulling the docker image, make sure you are logged in to registry.redhat.com using docker. Use the following command to login to registry.redhat.com. 
+Before pulling the docker image, make sure you are logged in to registry.redhat.io using docker. Use the following command to login to registry.redhat.io. 
```

Comment:
ensure you are logged in

Reply:
I will leave this change to be updated in this PR - https://github.com/ansible/aap-docs/pull/167 if that is ok

Reply:
I will revert the earlier change to allow it to be handled in #167